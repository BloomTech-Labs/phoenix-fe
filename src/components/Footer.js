import React from 'react';
import Terms from './Terms.js';
import '../styles/Footer.css';
import FacebookIcon from '@material-ui/icons/Facebook';
import TwitterIcon from '@material-ui/icons/Twitter';
import YouTubeIcon from '@material-ui/icons/YouTube';
import InstagramIcon from '@material-ui/icons/Instagram';
import { makeStyles } from '@material-ui/core';

const useStyles = makeStyles((theme)=> ({
    socialIcons: {
        color: theme.palette.primary.dark
    },
    copyright: {
        backgroundColor: theme.palette.secondary.dark
    }
}))

const Footer = () => {
    const classes = useStyles();

    return (
        <footer>
        <section className="top-footer">
            <div className="info">
                <div>
                    
                    <ul>
                    Your Account
                        <li>Sign up</li>
                        <li>Log in</li>
                        <li>Help</li>
                        <li><Terms/></li>
                    </ul>
                </div>
                <div>
                    <ul>
                    Discover
                        <li>Calendar</li>
                        <li>Topics</li>
                        <li>Cities</li>
                    </ul>
                </div>
                <div>
                    <ul>
                    Phoenix
                        <li>About</li>
                        <li>Phoenix Premium</li>
                        <li>Apps</li>
                        <li>Blog</li>
                    </ul>
                </div>
                <div className="follow">
                    <ul className={classes.socialIcons}>
                    <span style={{ color: "white" }}>Follow us</span>
                        <li><FacebookIcon /></li>
                        <li><TwitterIcon/></li>
                        <li><YouTubeIcon/></li>
                        <li><InstagramIcon/></li> 
                    </ul>
                    </div>
                </div>
        </section>
        <section className={classes.copyright}>
            <div className="copyright-text">
            <p>Made with ❤ © Lambda School 2020</p>
            </div>         
        </section>
        </footer>
    )
}

export default Footer;