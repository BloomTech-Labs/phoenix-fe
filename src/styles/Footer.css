footer {
  flex-shrink: 0;
  font-weight: bold;
  font-size: 16px;
  
}

section {
  background-color: #0b8b71;
  height: 275px;
}

.info {
  display: flex;
  max-width: 100%;
  justify-content: space-evenly;
  width: 1000px;
  color: #f4f0f0;
}
.info ul {
  list-style: none;
  justify-content: center;
  line-height: 3rem;
  padding: 0;
}
.follow img {
    width: 25px;
    height: 25px;
}

.terms-section {
  background-color: #07614f;
  text-align: left;
  justify-content: center;
  height: 250px;
}
.terms-section div {
  padding: 1rem 3rem;
  margin: 0 auto;
}
.copyright-text {
  height: 100%;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}
.top-footer {
  display: flex;
  align-items: center;
  justify-content: center;
}

@media (max-width: 768px) {
  .info {
    max-width: 90%;
    max-height: 5%;
    display: none;
  }

  .terms-section div {
    width: auto;
  }
}

@media (max-width: 500px) {
  .info {
    max-width: 90%;
    max-height: 1%;
    display: none;
  }
}
