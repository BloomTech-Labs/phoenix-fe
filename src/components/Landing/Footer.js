import React from 'react';
import Terms from './Terms.js';
import './../../styles/Footer.css';
import FacebookIcon from '@material-ui/icons/Facebook';
import TwitterIcon from '@material-ui/icons/Twitter';
import YouTubeIcon from '@material-ui/icons/YouTube';
import InstagramIcon from '@material-ui/icons/Instagram';
import { makeStyles } from '@material-ui/core';
import Button from "@material-ui/core/Button";

const useStyles = makeStyles((theme)=> ({
    socialIcons: {
        color: theme.palette.secondary.contrastText
    },
    copyright: {
        backgroundColor: theme.palette.secondary.dark
    }
}))

const Footer = () => {
    const classes = useStyles();

    return (
        <footer>
        <section className="top-footer">
            <div className="info">
                <div>
                    
                    <ul>
                    Your Account
                        <li>Sign up</li>
                        <li>Log in</li>
                        <li>Help</li>
                    </ul>
                </div>
                <div>
                    <ul>
                    Discover
                        <li>Calendar</li>
                        <li>Topics</li>
                        <li>Cities</li>
                    </ul>
                </div>
                <div>
                    <ul>
                    Phoenix
                        <li>About</li>
                        <li>Phoenix Premium</li>
                        <li>Apps</li>
                        <li>Blog</li>
                    </ul>
                </div>
                <div className="follow">
                    <ul className={classes.socialIcons} >
                    <span style={{ color: "white" }}>Follow us</span>
                        <li><FacebookIcon /></li>
                        <li><TwitterIcon/></li>
                        <li><YouTubeIcon/></li>
                        <li><InstagramIcon/></li> 
                    </ul>
                    </div>
                </div>
        </section>
        <section className={classes.copyright}>
            <div className="copyright-text">
                <Button style={{ marginLeft: "16px" }}>
                    <Terms />
                </Button>
            <p>Made with ❤ © Lambda School 2020</p>
            </div>         
        </section>
        </footer>
    )
}

export default Footer;