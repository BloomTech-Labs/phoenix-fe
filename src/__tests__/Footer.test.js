import React from 'react';
import renderer from 'react-test-renderer';
import ErrorBoundary from '../utils/ErrorBoundary.js';
import Footer from '../components/Footer.js';

it('renders the Footer correctly', () => {
    const tree = renderer.create(
        <ErrorBoundary>
            <Footer />
        </ErrorBoundary>
).toJSON();
  expect(tree).toMatchSnapshot();
});