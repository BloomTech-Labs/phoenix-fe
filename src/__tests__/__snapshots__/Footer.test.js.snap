// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Footer correctly 1`] = `
<footer>
  <section>
    <div
      className="info"
    >
      <div>
        Your Account
        <ul>
          <li>
            Sign up
          </li>
          <li>
            Log in
          </li>
          <li>
            Help
          </li>
        </ul>
      </div>
      <div>
        Discover
        <ul>
          <li>
            Calendar
          </li>
          <li>
            Topics
          </li>
          <li>
            Cities
          </li>
        </ul>
      </div>
      <div>
        Phoenix
        <ul>
          <li>
            About
          </li>
          <li>
            Phoenix Premium
          </li>
          <li>
            Apps
          </li>
          <li>
            Blog
          </li>
        </ul>
      </div>
      <div
        className="follow"
      >
        Follow us
        <ul>
          <li>
            <img
              alt="facebook app badges"
              src="facebook.png"
            />
          </li>
          <li>
            <img
              alt="twitter app badges"
              src="twitter.png"
            />
          </li>
          <li>
            <img
              alt="youtube app badges"
              src="youtube.png"
            />
          </li>
          <li>
            <img
              alt="instagram app badges"
              src="instagram.png"
            />
          </li>
        </ul>
      </div>
    </div>
  </section>
  <section
    className="terms-section"
  >
    <div>
      <div
        className="fullTerms"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "border": "none",
              "color": "#f4f0f0",
            }
          }
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Terms Of Service
          </span>
        </button>
      </div>
    </div>
  </section>
</footer>
`;

exports[`renders the Navbar correctly 1`] = `
<footer>
  <section>
    <div
      className="info"
    >
      <div>
        Your Account
        <ul>
          <li>
            Sign up
          </li>
          <li>
            Log in
          </li>
          <li>
            Help
          </li>
        </ul>
      </div>
      <div>
        Discover
        <ul>
          <li>
            Calendar
          </li>
          <li>
            Topics
          </li>
          <li>
            Cities
          </li>
        </ul>
      </div>
      <div>
        Phoenix
        <ul>
          <li>
            About
          </li>
          <li>
            Phoenix Premium
          </li>
          <li>
            Apps
          </li>
          <li>
            Blog
          </li>
        </ul>
      </div>
      <div
        className="follow"
      >
        Follow us
        <ul>
          <li>
            <img
              alt="facebook app badges"
              src="facebook.png"
            />
          </li>
          <li>
            <img
              alt="twitter app badges"
              src="twitter.png"
            />
          </li>
          <li>
            <img
              alt="youtube app badges"
              src="youtube.png"
            />
          </li>
          <li>
            <img
              alt="instagram app badges"
              src="instagram.png"
            />
          </li>
        </ul>
      </div>
    </div>
  </section>
  <section
    className="terms-section"
  >
    <div>
      <div
        className="fullTerms"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          style={
            Object {
              "border": "none",
              "color": "#f4f0f0",
            }
          }
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Terms Of Service
          </span>
        </button>
      </div>
    </div>
  </section>
</footer>
`;
